packages/web-app/src/modules/home-views/NavigationBarContainer.tsx

```diff
@@ -1,4 +1,5 @@
 import { Avatar, AvatarDefault, BonusCard, NavigationBar } from '@saladtechnologies/garden-components'
+import type { TargetRewardInfo } from '@saladtechnologies/garden-components/lib/components/NavigationBar/components/DesktopNavigationBar/TargetRewardStatus'
```

Comment:
doesn't index reexport this to the top?
```suggestion
import type { TargetRewardInfo } from '@saladtechnologies/garden-components'
```

Reply:
unfortunately the import you have suggested is not working